pkg/providers/github.go

```diff
@@ -68,28 +69,36 @@ func (p *GithubProvider) Validate(hook Hook) bool {
 }
 
 func (p *GithubProvider) GetProviderName() string {
-	return GithubName;
+	return GithubName
 }
 
 func (p *GithubProvider) GetCommitter(hook Hook) string {
+	eventType := Event(hook.Headers[XGitHubEvent])
 	var pushPayloadData GithubPushPayload
 	var pullRequestPayloadData GithubPullRequestPayload
-	if err := json.Unmarshal(hook.Payload, &pushPayloadData); err != nil {
-		log.Printf("Github payload unmarshaling failed for Push event: %v", err)
-		log.Printf("Now trying to unmarshal for pull request event")
-		if err = json.Unmarshal(hook.Payload, &pullRequestPayloadData); err != nil {
-			log.Printf("Github payload unmarshaling failed for pull request event: %v", err)
-			return ""
-		}
-	}
+	var issueCommentPayloadData GithubIssueCommentPayload
 
-	eventType := Event(hook.Headers[XGitHubEvent])
+	log.Printf("Received event type: %v", eventType)
 	switch eventType {
 	case GithubPushEvent:
+		if err := json.Unmarshal(hook.Payload, &pushPayloadData); err != nil {
+			log.Printf("Github payload unmarshaling failed for Push event: %v", err)
+			return ""
+		}
 		return pushPayloadData.Sender.Login
 	case GithubPullRequestEvent:
+		if err := json.Unmarshal(hook.Payload, &pullRequestPayloadData); err != nil {
+			return ""
```

Comment:
@jeremyplichtafc perhaps an error log here as well as to why pull marshaling of pull request even failed?

Reply:
good catch. got missed when I was reorganizing this section. fixed